packages/arb-avm-cpp/data_storage/src/messagestore.cpp

```diff
@@ -31,7 +31,7 @@
 nonstd::optional<rocksdb::Status> MessageStore::addMessages(
     const uint256_t first_sequence_number,
     const uint64_t block_height,
-    const std::vector<Tuple>& messages,
+    const std::vector<rocksdb::Slice>& messages,
     const std::vector<uint256_t>& inbox_hashes,
```

Comment:
I think it might make more sense to calculate these in here and only pass in the `previous_inbox_hash`